data/src/test/java/org/apache/iceberg/data/RandomGenericData.java

```diff
@@ -55,8 +55,8 @@ private RandomGenericData() {}
     return records;
   }
 
-  private static class RandomRecordGenerator extends RandomDataGenerator<Record> {
-    private RandomRecordGenerator(long seed) {
+  public static class RandomRecordGenerator extends RandomDataGenerator<Record> {
```

Comment:
This don't have to be public, see comment https://github.com/apache/iceberg/pull/1266/files#r464903356

Reply:
Fixed.